### PR TITLE
Update partial state card

### PR DESCRIPTION
**Pull request by keirog**
- only shows 1 partial card if user is following under 4 topics
- amends copy to encourage users to follow at least 4

**Time view**

Before | After
---|---
before-time | after-time

**Topic view**

Before | After
--- | ---
before-topic | after-topic

### CODE REVIEW

File: client/components/dashboard/main.scss [hunk 1]
```diff
@@ -40,21 +40,29 @@
 }
 
 .myft__dashboard-card--empty {
+	@include oGridRespondTo($until: L) {
+		min-width: 100%;
+	}
+
 	.n-topic {
+		align-items: center;
```
Comment (phamann) [inline client/components/dashboard/main.scss]: :heart_eyes: What is this dark magic flex box foo?

File: server/controllers/following/layout.js [hunk 2]
```diff
@@ -91,7 +91,7 @@ module.exports = (req, res, next) => {
 				referrerTracking: '#myft:my-news',
 				followed,
 				showEmpty,
-				emptySize: Array(showEmpty ? (4 - followed.length) : 0).fill(1),
+				emptySize: [parseInt(showEmpty, 10)],
```
Comment (phamann) [inline server/controllers/following/layout.js]: Nice!

**Comment by phamann** [comment]
:+1:

File: client/components/dashboard/main.scss (hunk 1 above)
Comment (keirog) [inline client/components/dashboard/main.scss]: A lot of flexbox seems mad, but this one I like :smile:

**Comment by keirog** [comment]
(Am ignoring the discussion about the plus symbol btw, in order to actually ship the thing)

**Pull request merged**